tests/integration/data/seed.py

```diff
@@ -43,3 +44,82 @@ def setup_list_data(idx, target_namespace, wait):
 
     if wait:
         poll_fetch_for_ids_in_namespace(idx, ids=["999"], namespace=target_namespace)
+
+
+def weird_invalid_ids():
+    invisible = [
+        "⠀",  # U+2800
+        " ",  # U+00A0
+        "­",  # U+00AD
+        "឴",  # U+17F4
+        "᠎",  # U+180E
+        " ",  # U+2000
+        " ",  # U+2001
+        " ",  # U+2002
+    ]
+    emojis = list("🌲🍦")
+    two_byte = list("田中さんにあげて下さい")
+    quotes = ["‘", "’", "“", "”", "„", "‟", "‹", "›", "❛", "❜", "❝", "❞", "❮", "❯", "＂", "＇", "｢", "｣"]
+
+    return invisible + emojis + two_byte + quotes
+
+
+def weird_valid_ids():
+    # Drawing inspiration from the big list of naughty strings https://github.com/minimaxir/big-list-of-naughty-strings/blob/master/blns.txt
+    ids = []
+
+    numbers = list("1234567890")
+    invisible = [" ", "\n", "\t", "\r"]
+    punctuation = list("!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~")
+    escaped = [f"\\{c}" for c in punctuation]
+
+    characters = numbers + invisible + punctuation + escaped
+    ids.extend(characters)
+    ids.extend(["".join(x) for x in itertools.combinations_with_replacement(characters, 2)])
+
+    boolean_ish = [
+        "undefined",
+        "nil",
+        "null",
+        "Null",
+        "NULL",
+        "None",
+        "True",
+        "False",
+        "true",
+        "false",
+    ]
+    ids.extend(boolean_ish)
+
+    script_injection = [
+        "<script>alert(0)</script>",
+        "<svg><script>123<1>alert(3)</script>",
+        '" onfocus=JaVaSCript:alert(10) autofocus',
+        "javascript:alert(1)",
+        "javascript:alert(1);",
+        '<img src\x32=x onerror="javascript:alert(182)">' "1;DROP TABLE users",
+        "' OR 1=1 -- 1",
+        "' OR '1'='1",
+    ]
+    ids.extend(script_injection)
+
+    unwanted_interpolation = [
+        "$HOME",
+        "$ENV{'HOME'}",
+        "%d",
+        "%s",
+        "%n",
+        "%x",
+        "{0}",
+    ]
+    ids.extend(unwanted_interpolation)
```

Comment:
total nit, but it just be simpler to do something like `return booleanish + script_injection + unwanted_interpolation`, instead of calling `ids.extend` mult times?